CanUr/Player.cs

```diff
@@ -0,0 +1,64 @@
+﻿using UnityEngine;
+using Assets.Pixelation.Example.Scripts;
+using System.Collections;
+
+public class Player : MonoBehaviour {
+	//REVOLUTIE VOOR DE HAAKJES PLZ!!!
+
+	public bool lockMouseMovement;
+	public MouseLook look;
+
+	public Transform weapon;
+
+    	public int health {
```

Comment:
Properties met hoofdletters graag

Reply:
Is het eigenlijk niet gek dat we properties ook hoofdletters maken? Uiteindelijk zijn het gewoon variabelen toch?
Puntje voor volgende week.